beatrix/src/test/resources/catalogs/testCatalogFixedTerm/fixedterm-and-biennial-billing.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2020-2020 Equinix, Inc
+  ~ Copyright 2014-2020 The Billing Project, LLC
+  ~
+  ~ The Billing Project licenses this file to you under the Apache License, version 2.0
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License.  You may obtain a copy of the License at:
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+  ~ License for the specific language governing permissions and limitations
+  ~ under the License.
+  -->
+<catalog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://docs.killbill.io/latest/catalog.xsd">
+    <effectiveDate>2019-01-01T00:00:00+00:00</effectiveDate>
+    <catalogName>ExampleCatalog</catalogName>
+    <recurringBillingMode>IN_ADVANCE</recurringBillingMode>
+    <currencies>
+        <currency>USD</currency>
+    </currencies>
+    <products>
+        <product name="p1">
+            <category>BASE</category>
+        </product>
+    </products>
+    <rules>
+        <changePolicy>
+            <changePolicyCase>
+                <policy>END_OF_TERM</policy>
+            </changePolicyCase>
+        </changePolicy>
+        <cancelPolicy>
+            <cancelPolicyCase>
+                <policy>END_OF_TERM</policy>
+            </cancelPolicyCase>
+        </cancelPolicy>
+    </rules>
+    <plans>
+        <plan name="p1-biennial">
```

Comment:
We typically name our plans `<productNameInLowerCase>-<billingPeriod>-<optionalOtherStuff>` as shown [here](https://github.com/killbill/killbill/blob/master/beatrix/src/test/resources/catalogs/default/catalogTest.xml#L509) to make it easier to understand what they are.

Reply:
Understood, in this case, the product is named `p1` and hence the plan name is `p1-biennial`. Do you want the product to be renamed to something more explanatory?

Reply:
My bad, what you have is great.

Reply:
I've now renamed the product to "pistol" to make it more explanatory and also renamed the plans accordingly